config/1es-pools/azsdk-pool-mms-win-2019-general/provisioning.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "00:00": 0
+    },
+    {
+        "16:00": 25
```

Comment:
Is there a reason to choose 25? As opposed to something like 10?

Reply:
I'm happy to choose 10. The balancing act here is the latency on builds starting. If there are agents available for a job, then you'll see it start in 0-30 seconds most of the time. If not you have to wait 300 seconds for an agent to spin up. If you get too large pipelines come in at the same time then your buffer is mostly exhausted and someone would notice queuing.

If they came in further apart you might not notice. So its a tuning thing.